lib/reply.js

```diff
@@ -130,6 +130,10 @@ Reply.prototype.send = function (payload) {
     return this
   }
 
+  if (this.request.raw && this.request.raw.aborted === true) {
+    return this
```

Comment:
Please run the onError hook with a new Error (create an ad-hoc
one inside lib/error.js). Note that if the payload is a stream you must call .destroy() on it too to avoid memory leaks.

Reply:
Are you sure you want to treat aborted requests as errors? Is it not a special case?
Currently when reply.send is called in this case with anything other than a stream there would be no error. Errors happen only with streams
For example if someone wants to handle aborted requests it is possible before calling reply.send

Thanks for the tip on destroy(). Do you think this would be a good way to destroy streams?
```
if (payload && payload.destroyed === false && typeof payload.destroy === 'function') {
  payload.destroy()
 }
```

Reply:
Then I think there is nothing we can do in this PR and the current behavior is correct.

Reply:
OK that's fine by me if you think so, I can fix this problem in my codebase using a decorator or an onSend hook. So was my description not clear about the problem at https://github.com/fastify/help/issues/607 ?